<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabTwo</class>
 <widget class="QWidget" name="LabTwo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="resultTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>851</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>进程号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>到达时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>服务时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>作业大小</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>优先级</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>剩余时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>状态</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>完成时间</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>371</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QPushButton" name="continueButton">
      <property name="text">
       <string>继续</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pauseButton">
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>时间片轮转</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>先来先服务</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>短作业优先</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>静态优先权有限</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>高响应比</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;算法选择:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>101</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="addProcessButton">
      <property name="text">
       <string>添加进程</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="loadDataButton">
      <property name="text">
       <string>载入数据</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="compareButton">
      <property name="text">
       <string>性能比较</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
